pulumi/consul-acl-policies/consul-agent.hcl

```diff
@@ -0,0 +1,19 @@
+# Policy used for consul agents, in particular nomad agents.
+
+# Allow the agent to register its own node in the Catalog and update its network coordinates, update node health checks
+# and have write access to its own agent config files.
+node "" {
```

Comment:
Are all these ACL rules supposed to be targeted to `""`? Is that some kind of wild-card?

Reply:
Yes, I'll add comments for this since its unusual